From ab7af2d0c57da04924b7c478e1970bdec514ce88 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jonas=20Dre=C3=9Fler?= <verdre@v0yd.nl>
Date: Wed, 11 Sep 2019 12:31:24 +0200
Subject: [PATCH 06/30] clutter/actor: Remove unused private function

`_clutter_actor_get_transition` is almost identical to
`clutter_actor_get_transition` and isn't used anywhere in the code (we
stopped using it with ee00e37b), remove it!

https://gitlab.gnome.org/GNOME/mutter/merge_requests/791
---
 clutter/clutter/clutter-actor-private.h |  3 ---
 clutter/clutter/clutter-actor.c         | 14 --------------
 2 files changed, 17 deletions(-)

diff --git a/clutter/clutter/clutter-actor-private.h b/clutter/clutter/clutter-actor-private.h
index 9bf1a3049..119ec4296 100644
--- a/clutter/clutter/clutter-actor-private.h
+++ b/clutter/clutter/clutter-actor-private.h
@@ -242,9 +242,6 @@ ClutterAnimationInfo *          _clutter_actor_get_animation_info
 ClutterTransition *             _clutter_actor_create_transition                        (ClutterActor *self,
                                                                                          GParamSpec   *pspec,
                                                                                          ...);
-ClutterTransition *             _clutter_actor_get_transition                           (ClutterActor *self,
-                                                                                         GParamSpec   *pspec);
-
 gboolean                        _clutter_actor_foreach_child                            (ClutterActor *self,
                                                                                          ClutterForeachCallback callback,
                                                                                          gpointer user_data);
diff --git a/clutter/clutter/clutter-actor.c b/clutter/clutter/clutter-actor.c
index bbfc43721..f600a838c 100644
--- a/clutter/clutter/clutter-actor.c
+++ b/clutter/clutter/clutter-actor.c
@@ -19466,20 +19466,6 @@ _clutter_actor_get_animation_info (ClutterActor *self)
   return res;
 }
 
-ClutterTransition *
-_clutter_actor_get_transition (ClutterActor *actor,
-                               GParamSpec   *pspec)
-{
-  const ClutterAnimationInfo *info;
-
-  info = _clutter_actor_get_animation_info_or_defaults (actor);
-
-  if (info->transitions == NULL)
-    return NULL;
-
-  return g_hash_table_lookup (info->transitions, pspec->name);
-}
-
 static void
 transition_closure_free (gpointer data)
 {
-- 
2.23.0

